### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-caasp.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Sep 27 10:03:25 UTC 2017 - jreidinger@suse.com
+
+- fix guilding with roles with order (related to fix for
```
Comment (imobachgs) [inline package/yast2-caasp.changes]: Is `guilding` the right word?

File: test/lib/widgets/ntp_server_test.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 describe Y2Caasp::Widgets::NtpServer do
   subject(:widget) { Y2Caasp::Widgets::NtpServer.new }
-  let(:dashboard_role) { ::Installation::SystemRole.new(id: "dashboard_role") }
+  let(:dashboard_role) { ::Installation::SystemRole.new(id: "dashboard_role", order: "100") }
```
Comment (imobachgs) [inline test/lib/widgets/ntp_server_test.rb:8]: Does exist a default value for `order`? (Just asking)

File: package/yast2-caasp.changes (hunk 1 above)
Comment (jreidinger) [reply package/yast2-caasp.changes]: it should be building :)

File: test/lib/widgets/ntp_server_test.rb (hunk 2 above)
Comment (jreidinger) [reply test/lib/widgets/ntp_server_test.rb:8]: no, there was discussion and intentionally without default

**Review by imobachgs — APPROVED** [review]

**Review by imobachgs — APPROVED** [review]

**Pull request merged**